snapcraft/_baseplugin.py

```diff
@@ -131,6 +131,10 @@ def clean_build(self):
         """Clean the artifacts that resulted from building this part."""
         pass
 
+    def get_manifest(self):
+        """Return the information to record after the build of this part."""
```

Comment:
Does the return value here need to be a dict? Documenting expectations here would be nice.